app/views/report/_proquest_thesis.html.erb

```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= link_to title_helper(proquest_thesis), edit_admin_thesis_path(proquest_thesis.id) %></td>
+  <td>
+    <% proquest_thesis.authors.each do |author| %>
+      <%= link_to author.user.display_name, edit_admin_author_path(author.id) %><br>
```

Comment:
This is super helpful. I've noticed some of our views don't link usefully to where you can view/edit the data driving the view.

The resulting administrate edit feels a bit weird though as it only has a checkbox which doesn't seem to fully account for the various statuses that we imply are available:

- yes, the user definitely opted-in
- no, the user definitely opted-out
- the user didn't edit the data at all so it's null

I think if a staff member goes to that form and then just clicks "save" the result will change from "null" no "no" implying a user _did_ make a choice which isn't accurate (whether that matters is unclear). Thoughts on whether we can have administrate more accurate reflect the yes/no/null options on that form?

Reply:
Yes, we should address this; null and false are very different values in this case. One potential solution could be to change the field to to a select, so that null would be one of the options in the drop-down menu.